linera-rpc/tests/snapshots/format__format.yaml.snap

```diff
@@ -1003,7 +1003,7 @@ SystemOperation:
       ReadBlob:
         STRUCT:
           - blob_id:
-              TYPENAME: BlobId
+              TYPENAME: BlobIdHelper
```

Comment:
This intermediate type now leaks to client-facing API. I think it's a bit of a code smell and we should try to retaing the `BlobId` one.

This is actually kinda funny b/c `BlobId` is a `pub struct` and it's _not_ in the API while `BlobIdHelper` is private (no `pub`) and _is_ in the API :)

Reply:
That _is_ weird 🤔

Reply:
This issue was solved by @afck's suggestion.